Dockerfile

```diff
@@ -21,8 +21,8 @@ RUN APT_INSTALL="apt-get install -y --no-install-recommends" && \
         add-apt-repository ppa:deadsnakes/ppa && \
         apt-get update && \
         DEBIAN_FRONTEND=noninteractive $APT_INSTALL \
-            python3.6 \
-            python3.6-dev \
+            python3.8 \
+            python3.8-dev \
```

Comment:
When updating this Dockerfile, I planned to switch to the newer Ubuntu 20.04 image. As a bonus, you don't need to install Python 3.8 because it comes with the new Ubuntu.

Reply:
Ok, we'll change it later, but I don't want to handle switching to Ubuntu 20.04 in this PR.